### PR TITLE
fix Open Graph og:image link

### PR DESCRIPTION
**Pull request by ashimokawa**
This fixes the Open Graph og:image link, which fix fix twitter preview cards, among others

you can see the problem by using this link, and then clicking on the "Open Graph" button:

http://debug.iframely.com/?uri=https://try.gitea.io

Or use the twitter card validator

https://cards-dev.twitter.com/validator

Corresponding Codeberg issue: https://codeberg.org/Codeberg/Community/issues/39

### CODE REVIEW

File: templates/base/head.tmpl
Comment (lafriks) [inline templates/base/head.tmpl]:
```suggestion
	<meta property="og:image" content="{{AppUrl}}/img/gitea-lg.png" />
```
AppUrl already contains subpath

**Review by lafriks — DISMISSED** [review]

File: templates/base/head.tmpl
Comment (ashimokawa) [reply templates/base/head.tmpl]: Thanks, makes sense ;)

**Review by jonasfranz — APPROVED** [review]

**Review by kolaente — APPROVED** [review]

**Comment by techknowlogick** [comment]
@ashimokawa Please check "allow edits from maintainers" so we can merge latest changes from master before merging.

**Comment by ashimokawa** [comment]
> @ashimokawa Please check "allow edits from maintainers" so we can merge latest changes from master before merging.

Done,
also rebased to latest master, feel free to squash the two commits

**Pull request merged**
